scripts/system/create/entitySelectionTool/entitySelectionTool.js

```diff
@@ -2801,17 +2907,17 @@ SelectionDisplay = (function() {
                 }
                 
                 if (direction === ROTATE_DIRECTION.PITCH) {
-                    rotationNormal = { x: 1, y: 0, z: 0 };
+                    rotationNormal = { x: 0, y: 1, z: 0 };
                     worldRotation = worldRotationY;
-                    selectedHandle = handleRotatePitchRing;
+                    selectedHandle = handleRotateGreenRing;
                 } else if (direction === ROTATE_DIRECTION.YAW) {
-                    rotationNormal = { x: 0, y: 1, z: 0 };
+                    rotationNormal = { x: 0, y: 0, z: 1 };
                     worldRotation = worldRotationZ;
-                    selectedHandle = handleRotateYawRing;
+                    selectedHandle = handleRotateBlueRing;
                 } else if (direction === ROTATE_DIRECTION.ROLL) {
-                    rotationNormal = { x: 0, y: 0, z: 1 };
+                    rotationNormal = { x: 1, y: 0, z: 0 };
                     worldRotation = worldRotationX;
-                    selectedHandle = handleRotateRollRing;
+                    selectedHandle = handleRotateRedRing;
                 }
```

Comment:
This seems a little odd. ... Pitch should normally be associated with rotation about the x-axis and red colour, yaw the y-axis and green, and roll the z-axis and blue. ... Similarly throughout the PR changes.

Reply:
I reset those changes in my next commit and found the right approach to address this.